pkg/core/region.go

```diff
@@ -1471,22 +1471,33 @@ func (r *RegionsInfo) QueryRegions(
 	keys, prevKeys [][]byte, ids []uint64, needBuckets bool,
 ) ([]uint64, []uint64, map[uint64]*pdpb.RegionResponse) {
 	// Iterate the region keys to find the regions.
+	queryRegionKeysCount.Add(float64(len(keys)))
+	start := time.Now()
 	regions := r.getRegionsByKeys(keys)
+	queryRegionByKeysDuration.Observe(time.Since(start).Seconds())
 	// Assert the returned regions count matches the input keys.
 	if len(regions) != len(keys) {
 		panic("returned regions count mismatch with the input keys")
 	}
+
 	// Iterate the prevKeys to find the regions.
+	queryRegionPrevKeysCount.Add(float64(len(prevKeys)))
+	start = time.Now()
 	prevRegions := r.getRegionsByPrevKeys(prevKeys)
+	queryRegionByPrevKeysDuration.Observe(time.Since(start).Seconds())
```

Comment:
If there is no prevKeys, do we still need to record its time?

Reply:
Fixed.

<img width="1723" src="https://github.com/user-attachments/assets/1ed457f9-8dce-4c47-ba4b-c6501f587494" />